msteams-platform/messaging-extensions/high-quality-message-extension.md

```diff
@@ -751,16 +751,77 @@ For a plugin to be validated, invoked, and work seamlessly, ensure that it meets
+|Microsoft 365 Channel| For users to interact with your message extension from Outlook, you need to add Microsoft 365 channel to your bot. For more information, see [Add Microsoft 365 channel](../m365-apps/extend-m365-teams-message-extension.md#add-microsoft-365-channel-for-your-app). [*Mandatory*]|
 | Single sign-on (SSO) | If applicable, update your Microsoft Entra ID app registration for SSO.  [*Recommended*] |
 | Content Security Policy |If applicable, modify your Content Security Policy headers. [*Recommended*] |
 
 > [!IMPORTANT]
 > If applicable, update your Content Security Policy headers and `X-Frame-Options` in accordance with [Configure Content Security Policy headers](../m365-apps/extend-m365-teams-personal-tab.md#configure-content-security-policy-headers).
 
+## Message extensions Plugins in Copilot for Microsoft 365
+
+Copilot extensions customize and extend the Copilot for Microsoft 365 experience by bringing more skills and knowledge to Copilot for a personalized user experience. Using plugins, which are a specific type subset of Copilot extensions, users can integrate additional capabilities into Copilot by interacting with third-party applications, whether for retrieving or modifying information within those apps. For instance, message extension plugins facilitate searching for data in other applications so that Copilot can present it upon request when the plugin is activated.
+
+ If you've developed a plugin for Copilot in Teams or copilot.microsoft.com, you're already aware of the benefits it offers to users within their workflow. 
+ <!--To extend your plugin's functionality to Copilot in Word, Excel, PowerPoint, and OneNote, refer to the following document for the required additional steps.-->
+
+### Requirements for plugins in Word, PowerPoint, Excel, OneNote Copilots
+
+To ensure your plugins work with Word, Excel, PowerPoint, and OneNote Copilots, follow these requirements:
+
+* <b>Update Microsoft Azure Active Directory (Azure AD) app registration for SSO</b>
+
+    Azure Active Directory (AD) single sign-on (SSO) for message extensions work in the same way [as it does in Teams](../bots/how-to/authentication/auth-aad-sso-bots.md) or Outlook. However, you need to add the Office app Copilot’s client application identifier to the Azure AD app registration of your bot in your tenant's App registrations portal.
+
+    1. Sign in to [Azure portal](https://portal.azure.com/)  with your sandbox tenant account. 
+    1. Open <b>App registrations</b>. 
+    1. Select the name of your application to open its app registration. 
+    1. Select Expose an <b>API</b> (under Manage). 
+    1. In the <b>Authorized client applications</b> section, ensure the following Client Id values are listed:
+
+        | Microsoft 365 client application  | Client ID |
+        | --- | --- |
+        | OfficeAIAppChatCopilotExt | 3068386c-7a16-4f6a-a664-043b6b232816 |
```

Comment:
Can we rename `OfficeAIAppChatCopilotExt` to `Word, PowerPoint, Excel (Web, Desktop)`